API.yaml

```diff
@@ -1442,8 +1442,8 @@ components:
         description:
           description: Describe the output provider's features
           type: string
-        responseType:
-          description: Type of data returned by this output. Serve as a hint to determine what kind of view should be use for this output (ex. XY, Time Graph, Table, etc..)
+        providerType:
```

Comment:
To be consistent I think it should be outputType. As a bonus we can use an enum from open API to provide the values

Reply:
I'll update it